drivers/classic/src/initialize_atmos.c

```diff
@@ -101,39 +100,44 @@ initialize_atmos(atmos_data_struct    *atmos,
     double                     delta_t_plus;
     int                        have_dewpt;
     int                        have_shortwave;
-    int                        hour_offset_int;
-    int                        tmp_starthour, tmp_endhour;
-    short                      local_startyear;
-    short                      local_startmonth;
-    short                      local_startday;
-    short                      local_starthour;
-    unsigned short             day_in_year, year, month, days_in_month;
+    int                        sec_offset_int;
```

Comment:
Yep - I see your comments. You just don;t get an email copy of your own, but they are online and emailed to others. Joe is getting all of them ;-). There is probably an account preference where you can select to get an email copy of your own comments.


> On Jan 12, 2015, at 8:10 PM, Ted Bohn <notifications@github.com> wrote:
> 
> In drivers/classic/src/initialize_atmos.c:
> 
> > @@ -101,39 +100,44 @@ initialize_atmos(atmos_data_struct    *atmos,
> >      double                     delta_t_plus;
> >      int                        have_dewpt;
> >      int                        have_shortwave;
> > -    int                        hour_offset_int;
> > -    int                        tmp_starthour, tmp_endhour;
> > -    short                      local_startyear;
> > -    short                      local_startmonth;
> > -    short                      local_startday;
> > -    short                      local_starthour;
> > -    unsigned short             day_in_year, year, month, days_in_month;
> > +    int                        sec_offset_int;
> 
> By the way, I'm also reviewing the code (Joe asked me to). But I'm not seeing my comments show up in this email thread, strangely. Are you seeing my comments? Ted
> …
> —
> Reply to this email directly or view it on GitHub.
>